ga4gh/serverconfig.py

```diff
@@ -59,6 +61,13 @@ class ProductionConfig(BaseConfig):
     DATA_SOURCE = None
 
 
+class SimulatedConfig(BaseConfig):
```

Comment:
Here is the simulated config as its own PR https://github.com/ga4gh/server/pull/1017